tools/cli/src/mentat_cli/repl.rs

```diff
@@ -62,7 +56,8 @@ impl Repl {
         let mut input = InputReader::new();
 
         loop {
-            let res = input.read_input(if more.is_some() { MORE_PROMPT } else { DEFAULT_PROMPT });
+            let res = input.read_input(more.clone());
```

Comment:
I'm surprised you need this `clone`, since you're joining strings in `read_input`, not re-using them.